custom_components/opnsense/__init__.py

```
@@ -258,12 +259,25 @@ async def async_remove_entity(self, entity):
-        return None
+    def __init__(
+        self,
+        config_entry: ConfigEntry,
+        coordinator: OPNsenseDataUpdateCoordinator,
+        unique_id_suffix: str,
+        name_suffix: str | None = None,
+    ) -> None:
+        self.config_entry: ConfigEntry = config_entry
+        self.coordinator: OPNsenseDataUpdateCoordinator = coordinator
+        self._attr_unique_id: str = slugify(
+            f"{self.opnsense_device_unique_id}_{unique_id_suffix}"
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)